### ISSUE TITLE
Constants aren't working in staging

### ISSUE THREAD

**Issue by pulkitsinghal** [ShoppinPal/warehouse#26]
Errors such as the following pop-up in the Chrome browser's error console when testing the staging env:

```
GET http://shoppinpal.xxx.com/@@baseUrl@@loopbackApiRoot/UserModels/reportModels 404 (Not Found)
```

Both `baseUrl` and `loopbackApiRoot` are present in the `/client/app/scripts/shoppinpal-constants.js` file  and it is the responsibility of the build process to replace them with appropriate values from a `NON-git-tracked` file at `/server/boot/config.staging.json` ... so the CI server (Jenkins?) needs to play a role here and `Gruntfile.js` too.

**Comment by pulkitsinghal** [comment]
I'll be taking over for now.

**Comment by pulkitsinghal** [comment]
resolved in 6537e4d7aa1690495c6fee4d2c4b0bf3d1e25003
